nixos/modules/misc/ids.nix

```diff
@@ -275,6 +275,8 @@
       gocd-server = 252;
       terraria = 253;
       mattermost = 254;
+      quagga = 255;
```

Comment:
Not all daemons use static ids. Unless there's a specific reason to use static ids, please switch to dynamic ids.